test/page_object/base_page/header.js

```diff
@@ -2,9 +2,12 @@ const Element = require("../base_elements/base_elements");
 
 class Header {
   constructor() {
-    this.Support = new Element("xpath", "//*[@id=\"apc-megamenu\"]/li[3]/a");
-    this.HowToBuy = new Element("xpath", "//*[@id=\"apc-megamenu\"]/li[4]/a");
-    this.ProductSelector = new Element("xpath", "//a[contains(text(),'Product Selectors')]");
+    this.Support = new Element("xpath", '//*[@id="apc-megamenu"]/li[3]/a');
```

Comment:
Please add element name to your elements names:
`this.SupportButton`
`this.HowToBuyButton`, etc.